backend/src/postsDataSource.ts

```diff
@@ -1,5 +1,8 @@
 import { DataSource } from 'apollo-datasource'
 import { v4 as uuidv4 } from 'uuid'
+require('dotenv').config()
```

Comment:
Why do you `require('dotenv').config()` so often? I would expect only one file to require it and this file gets imported wherever needed.